Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="vlados.soc@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt

```

Comment:
Consider using a multi-stage build or cleaning up the package cache after installing dependencies to reduce the image size. You can do this by adding `&& rm -rf /var/lib/apt/lists/*` after the `pip install` command.